website/content/api-docs/secret/pki.mdx

```diff
@@ -2116,6 +2123,42 @@ $ curl \
 }
 ```
 
+<a name="read-cert-metadata"></a>
+
+### Read metadata
+
+This endpoint retrieves the metadata associated with the certificate
+specified by the serial number.
+
+| Method | Path                                 | Format |
+| :----- | :----------------------------------- |:------ |
+| `GET`  | `/pki/cert-metadata/:serial`         | JSON   |
+
+#### Parameters
+
+- `serial` `(string: <required>)` - Specifies the serial of the certificate
+whose metadata to read. This is part of the request URL.
+
+#### Sample request
+
+```shell-session
+$ curl \
```

Comment:
As this API requires authentication the curl command needs the following
 
`--header "X-Vault-Token: ..." \`